README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
+ sudo apt install gradle
+- Install Canberra
+ Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️What is Gradle
+
+Gradle is a automated build system that makes working with large projects easier.
+It streams line the build process by skipping re-execution and allows developers to preform tasks suing one line of code in the command line.
+
+
+
```

Comment:
It seems like **this** is the place to **actually show the reader how to build the ImageLab product**.

Scanning ahead, I don't see any clearly identified instruction on how to build the product.
That should be prominent, obvious, and easy to find.